pytorch_lightning/accelerators/accelerator.py

```diff
@@ -228,7 +228,7 @@ def validation_step_end(self, output: Optional[STEP_OUTPUT]) -> Optional[STEP_OU
         """
         return self.training_type_plugin.validation_step_end(output)
 
-    def backward(self, closure_loss: Tensor, *args: Any, **kwargs: Any) -> Tensor:
+    def backward(self, closure_loss: Tensor, optimizer: torch.optim.Optimizer, *args: Any, **kwargs: Any) -> Tensor:
```

Comment:
Does Optimizer vs torch.optim.Optimizer the same here?

Reply:
yes, we should choose just "Optimizer" here for consistency with the other signatures. It does not make a functional difference :)